src/styles/images/index.md.njk

```diff
@@ -4,26 +4,60 @@ description: Only use images if there’s a real user need
 section: Styles
 backlog_issue_id: 70
 layout: layout-pane.njk
+show_page_nav: true
 ---
 
 {% from "_example.njk" import example %}
 
 Avoid unnecessary decoration. Only use images if there’s a real user need.
 
-## Aspect ratio
+Make sure any information contained in an image can be understood by someone who cannot see it.
 
-### Photography
+## Photography
 
-The aspect ratio for photographs should be 3:2.
+Use photography when it’s important to show a lifelike representation of something. For example, showing a photo of a document a user has uploaded.
 
-{{ example({group: "styles", item: "images", example: "default", html: false, open: false, size: "l"}) }}
+Don't use photography to represent abstract concepts, or use clichéd stock photography.
+
+## Illustrations
+
+Use illustrations to show a simplified image of something complex. For example:
+- to demonstrate a physical interaction, like scanning a passport
+- to help users find something on a document, like a reference number
+
+{{ example({group: "styles", item: "images", example: "illustration", html: false, open: false, size: "l"}) }}
+
+### Creating illustrations
+
+Use as few visual elements as possible to communicate your message.
+
+Start by using the [GOV.UK colour palette](/styles/colour/#colour-palette). If you need more colours, try to use tints of the colour palette. 
+
+Use flat blocks of colour. Avoid using shadows or gradients, except when needed to communicate layering or depth.
```

Comment:
The example of a Biometric Residence Permit has a gradient... should we try to mention?
```suggestion
Use flat blocks of colour. Avoid using shadows or gradients, except when needed to show a distinctive feature of an object or to communicate layering or depth.
```

Reply:
Yeah sounds good